src/js/jsx/sections/export/ExportPanel.jsx

```diff
@@ -229,8 +222,26 @@ define(function (require, exports, module) {
                                 onClick={addAssetClickHandler || _.noop}
                                 onDoubleClick={this._addAssetDoubleClickHandler}>
                                 <SVGIcon
-                                    viewbox="0 0 12 12"
-                                    CSSID="plus" />
+                                    viewbox="0 0 16 16"
+                                    CSSID="add-new" />
+                            </Button>
+                            <Gutter />
+                            <Button
+                                className="button-iOS"
+                                title={strings.TOOLTIPS.EXPORT_IOS_PRESETS}
+                                onClick={addIOSAssetClickHandler || _.noop}>
```

Comment:
Pretty sure these `|| _.noop`s are not necessary. If it's null or undefined, React does the right thing.